terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/exceptionhandling/ExceptionHandlingTest.java

```diff
@@ -69,6 +69,7 @@ public void test01_02_requestControllerHandling() {
 
         // INFO Level Log
         dbLogProvider.waitForAssertion();
+        assertThat(driver.getTitle(), is("terasoluna-gfw-functionaltest"));
         assertThat(dbLogProvider.countContainsMessageAndLevelsAndLogger(
                 "\\[w.xx.0002\\] ResultMessages \\[type=info, list=\\[ResultMessage \\[code=i.gt.me.0004, args=\\[\\], text=null\\]\\]\\]",
                 "INFO", "org.terasoluna.gfw.common.exception.ExceptionLogger"),

```

Comment:
@btoutnp 
- [ ] The cause of the Travis error is that in some test cases the check at `assertThat(driver.getTitle()` is missing.
For all of test cases using DB log assertion you need to assert title.

For example, https://github.com/terasolunaorg/terasoluna-gfw-functionaltest/blob/2ae40165f3103002b06630c9f037d9f138a8cdd1/terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/exceptionhandling/ExceptionHandlingTest.java#L95-L96